### PR TITLE
feat(azure gcp) Highlight tags and backlink to docs

### PR DESCRIPTION
**Pull request by felixzieger**

### CODE REVIEW

File: docs/meshstack.gcp.landing-zones.md [hunk 1]
```diff
@@ -51,17 +51,21 @@ If the URL is changed or the underlying template updated the projects will autom
...
-| projectIdentifier  | The project identifier                                     |
-| projectId          | The ID of the GCP project associated with this meshProject |
+The properties of the provided configuration file will be expanded with properties from meshcloud and these can be used inside the template itself. The following properties are provided:
 
-In addition, any payment settings, project tags or customer tags are also provided to the template.
-For example, a tag named `myCustomerLabel` would be provided as a property with name `mesh-tag-my-customer-label`.
+| Template Property          | Description                                                                       |
+| ---------------------------| :---------------------------------------------------------------------------------|
+| customerIdentifier         | Customer Identifier                                                               |
+| tagCostCenter              | ID of the CostCenter defined for this meshProject.                                |
```
Comment (JohannesRudolph) [inline docs/meshstack.gcp.landing-zones.md:61]: f: I think tagCostCenter vs. mesh-tag-my-customer-label may not be correct (also notice different casing/kebab case etc.). Can you double check this?

File: docs/meshstack.gcp.landing-zones.md [hunk 2]
```diff
@@ -51,17 +51,21 @@ If the URL is changed or the underlying template updated the projects will autom
...
+| projectIdentifier          | The project identifier                                                            |
+| projectId                  | The ID of the GCP project associated with this meshProject                        |
+| mesh-tag-my-customer-label | Example for a  [MetaData Tag](./meshstack.tag-schema.md) named `mycustomerLabel`  |
+
+As the example `mesh-tag-my-customer-label` in the above table indicates, any payment settings, project tags or customer tags are also provided to the template.
+In the example, a tag named `myCustomerLabel` would be provided as a property with name `mesh-tag-my-customer-label`.
+See [MetaData Tags](./meshstack.tag-schema.md) for more information.
```
Comment (JohannesRudolph) [inline docs/meshstack.gcp.landing-zones.md:68]: d: I think @chewbeccy wants to make this `meshTags` consistently across the product. Until then the docs refer to them as `metadata tags` (not MetaData`

File: docs/meshstack.azure.landing-zones.md [hunk 3]
```diff
@@ -32,17 +32,21 @@ Blueprints must reside inside a Management Group. It is assumed it is in the sam
...
-For example, the value of the tag `costCenter` will be made available via the key `tagCostCenter`.
+| Parameter          | Description                                                                               |
+|--------------------|:------------------------------------------------------------------------------------------|
+| customerIdentifier | Customer Identifier                                                                       |
+| ~~costcenter~~     | ID of the CostCenter defined for this meshProject. (Deprecated. Please use tagCostCenter) |
+| projectIdentifier  | The project identifier                                                                    |
+| subscriptionId     | The ID of the Azure Subscription associated with this meshProject                         |
+| tagCostCenter      | Example for a  [MetaData Tag](./meshstack.tag-schema.md) named `costCenter`               |
+
+As the example `tagCostCenter` in the above table indicates, any payment settings, project tags or customer tags can also be used in the Blueprints. These parameter keys will have the prefix `tag`.
```
Comment (JohannesRudolph) [inline docs/meshstack.azure.landing-zones.md:47]: d: check if they not only have prefix, I think there's also a "name normalization" rule that we should explain (i.e. prepend tag, first letter is uppercase)

File: docs/meshstack.gcp.landing-zones.md (hunk 2 above)
Comment (chewbeccy) [reply docs/meshstack.gcp.landing-zones.md:68]: ok so lets keep them as metadata tags and I will change this for the whole docs in a another PR to get naming consistent

**Pull request merged**